src/screens/FeedScreen/ContentPoll.js

```diff
@@ -333,7 +334,7 @@ const stylesComponent = (pollLength) =>
     },
     voteFont: {
       fontSize: normalizeFontSize(12),
-      color: COLORS.blackgrey,
+      color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.blackgrey` to `#828282`. Ensure that this hex color is the intended color.
```diff
- color: COLORS.blackgrey
+ color: '#828282'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->